src/routes/boot.ts

```diff
@@ -420,6 +420,7 @@ const getUser = (
       'followingEmail',
       'followNotifications',
       'subscriptionFlags',
+      'defaultFeedId',
```

Comment:
also add to `UpdateUserInput` so we can set it on frontend?

Reply:
Added, but no specific test yet